src/modules/edit.js

```diff
@@ -0,0 +1,43 @@
+// added the edit function so it changes the local storage description
+const call = (index, value) => {
+  const retrieveData = localStorage.getItem('list');
+  const data = JSON.parse(retrieveData);
+
+  for (let i = 0; i < data.length; i += 1) {
+    if (data[i].index === index) {
+      data[i].description = value;
+      localStorage.setItem('list', JSON.stringify(data));
+    }
+  }
+};
+// when double clicked on the li the icon appears
+const removes = (index, li, work, icon) => {
+  li.style.backgroundColor = 'antiquewhite';
+  work.style.backgroundColor = 'antiquewhite';
+  icon.setAttribute('class', 'show');
+  icon.addEventListener('click', () => {
+    const retrieveData = localStorage.getItem('list');
+    const data = JSON.parse(retrieveData);
+    for (let i = 0; i < data.length; i += 1) {
+      if (data[i].index === index) {
+        data.splice(i, 1);
```

Comment:
We shall practice using array methods that don't mutate the original array such as [concat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat), [filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter), [slice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice) instead of push, pop, splice and so on. instead of [push](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push), [pop](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/pop), [splice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice) and so on.